README.md

```diff
@@ -37,3 +38,48 @@ Command | Usage
 /help | Brings up a quick menu to remind you of commands
 
 ## Setup
+1. Download the zip file from the green "Code" button above
+
+2. Extract the zip file into a folder
+3. Create a ```.txt``` file, but change its name to just ```.env```! 
+    3. **Note that there is NO name in front of  ```.env```**
+4. Open up the newly created ```.env``` file using Notepad or another of your choice
+5. Follow the ```Creating a Bot Account``` on this site:
+```https://discordpy.readthedocs.io/en/latest/discord.html```
+6. Copy the token from step 7 (of the site)
+7. Within ```.env```, you will have to add in **THREE** lines:
```

Comment:
Only `DISCORD_TOKEN` is required, other settings have defaults and are therefore optional.